commit 9bf7e24bf045cbe1d5caa447c49ee8babd8d2501 (refs/changes/04/3804/2)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2018-04-30T10:40:04-07:00 (1 year, 5 months ago)
    
    MANTA-3621 Mako error logs are big and have low SNR

diff --git a/sapi_manifests/mako/template b/sapi_manifests/mako/template
index c5775c7..aaafd9c 100644
--- a/sapi_manifests/mako/template
+++ b/sapi_manifests/mako/template
@@ -13,7 +13,7 @@ thread_pool mpu threads=8;
 {{/MAKO_THREAD_POOL_SIZE}}
 
 
-error_log /var/log/mako-error.log info;
+error_log /var/log/mako-error.log notice;
 
 events {
 	worker_connections  1024;
